### DISCUSSION TITLE
[next/image]: add svg placeholder option

### DISCUSSION THREAD

**Discussion by ecklf** [vercel/next.js#26489] [Ideas]
### Describe the feature you'd like to request

It would be awesome to have a traced SVG placeholder like `gatsby-image` for the next.js image component.

Example: https://using-gatsby-image.gatsbyjs.org/traced-svg/

### Describe the solution you'd like

Additional option for `placeholder` being `trace`
Required property for non-static files `traceDataURL` as SVG data.

### Describe alternatives you've considered

Not using `next/image` for this use-case :(

**Comment by evelynhathaway** [comment]
There is a third party package for this, [next-image-trace-loader](https://github.com/BrandonKlotz/next-image-trace-loader), but it'd be awesome to see this in the Next.js Image component. The `placeholder` API looks like it was built to come back to this someday.
